lanai/app.py

```diff
@@ -3,16 +3,10 @@
 
 
 class Lanai(object):
-    _default_config = dict(
-        host='0.0.0.0',
-        port=1234
-    )
     protocol_rule = dict()
 
-    def __init__(self, config=_default_config,
-                 connection_handler_class=ConnectionHandler):
-        self.config = config
+    def __init__(self, connection_handler_class=ConnectionHandler):
```

Comment:
How about `connection_handler_class` add in config?

Like this..
http://www.django-rest-framework.org/api-guide/settings/